engine/src/model.js

```diff
@@ -80,7 +80,7 @@ class Model {
 
       if (sectionPages.length > 0) {
         if (section) {
-          const isRepeatable = sectionPages.find(page => page.pageDef.repeatField)
+          const isRepeatable = sectionPages.find(page => page.pageDef.repeatField || page.pageDef.isRepeatable)
```

Comment:
This is creating complexity, where a section is considered repeatable if any of it's pages are repeatable. What if one of it's pages isn't? Or two pages are repeatable on different properties?

A section creates a nice container for repetition, "Dependent" becomes "Dependent 1" with any number of pages within it. We can also achieve the same thing as having a single repeatable page by simply putting that page in it's own section. This simplicity will reducee errors both when designing a form and when coding this functionality, so I'd like to see us change it - moving these page level properties to section.

Note also, because I'm on a roll - `isRepeatable` is a little confusing to me. A section with `repeatField` configured is also repeatable, but it has a different meaning if only `isRepeatable` is specified.

Reply:
1. I agree

2. > Note also, because I'm on a roll - isRepeatable is a little confusing to me. A section with repeatField configured is also repeatable, but it has a different meaning if only isRepeatable is specified.

yeah I know (hence draft PR) whilst I'm deciding how to approach..! I've set a task for myself, I mentioned in my op 
>  should be changed and unified. bool = isRepeatable behaviour (unspecified n), field name (string) = repeatField behaviour

thanks!